controller/src/beerocks/master/tasks/optimal_path_task.cpp

```diff
@@ -362,9 +362,11 @@ void optimal_path_task::work()
          * There is a high probability to get responsiveness less than 100% and therefore, the
          * threshold was changed to 50% as part of demo optimizations.
          * This hardcoded threshold is temporary and shall be revised in the future. 
+         * update: increasing to 80% since in bandsteering scenario there are only 2 bands
```

Comment:
please use `constexpr` and all this description can be moved to where it is defined